js/offer.js

```diff
@@ -0,0 +1,80 @@
+import {getHouseRentalCount} from './random-object.js';
+const cardTemplate = document.querySelector('#card').content.querySelector('.popup');
+const templateFRAGMENT = document.createDocumentFragment();
+const announcementMap = document.querySelector('.map__canvas');
+
+const mapTypeHousing = {
```

Comment:
now your name is like name of the function, but it is object, so name should be an noun, f. e. houseTypeMap